engine/runtime/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -278,6 +278,8 @@ object ProgramExecutionSupport {
     ctx: RuntimeContext
   ): PartialFunction[Throwable, Api.ExecutionResult.Diagnostic] = {
     case ex: AbstractTruffleException
+        // The empty language is allowed because `getLanguage` return null when
```

Comment:
```suggestion
        // The empty language is allowed because `getLanguage` returns null when
```

```suggestion
        // The empty language is allowed because `getLanguage` may return null when
```